opentimelineio_contrib/adapters/advanced_authoring_format.py

```diff
@@ -164,11 +186,23 @@ def _transcribe(item, parent=None, editRate=24, masterMobs=None):
     elif isinstance(item, aaf.component.SourceClip):
         result = otio.schema.Clip()
 
-        # ref = item.resolve_ref()
-        # name = ref.name
+        mobs = [item.resolve_ref()]
+        start = item.start_time
+
+        for c in item.walk():
+            mob = c.resolve_ref()
+            start += c.start_time
+            if mob:
+                mobs.append(mob)
+
+        # Evidently the last mob is the one with timecode
+        timecode = _extract_timecode(mobs[-1])
 
         length = item.length
         startTime = int(metadata.get("StartTime", "0"))
+        if timecode:
+            startTime += timecode
```

Comment:
Is `timecode` an int?

Reply:
Seems to be, yes.

Reply:
So maybe `media_start` or something would be a more clear name?

Reply:
You mean for the timecode variable?  or for startTime?  I figured timecode would be good since its what is getting pulled out of the timecode track.